src/main/java/org/jitsi/jigasi/AbstractGatewaySession.java

```diff
@@ -262,7 +262,7 @@ void notifyMemberJoined(ChatRoomMember member)
      * @param member the member who left the JVB conference
      */
     // FIXME: 17/07/17 JvbConference does not yet call this method
-    void notifyMemberLeft(ChatRoomMember member)
+    void notifyChatRoomMemberLeft(ChatRoomMember member)
```

Comment:
Unrelated to this PR, but: why do we not update `this.participantsCount` here? It seems to only increase

Reply:
I don't know, it's supposed to be global:

```
    /**
     * Global participant count during this session including the focus.
     */
    private int participantsCount = 0;
```

Reply:
I see